share/man/man7/hier.7

```diff
@@ -479,7 +479,7 @@ Z file system utilities
 .Pp
 .It Pa local/
 local executables, libraries, etc, installed by
-.Xr pkg 7
+.Xr pkg 8
 or
 .Xr ports 7
 .Pp
```

Comment:
```suggestion
(including use of the package management system for
.Xr ports 7
)
.Pp
```

The current edition implies that for installations, ports(7) is an alternative to pkg. Related misunderstandings are commonplace.

From <https://man.freebsd.org/cgi/man.cgi?query=ports&sektion=7&manpath=freebsd-current#TARGETS>: 

> … Usually, you will only use the install target. 

– and: 

> Install the port and register	it with the package system.

That is, the package management system. From Warren Block's [What is pkg on FreeBSD?](http://www.wonkity.com/~wblock/docs/html/pkg.html) (2014) point 4: 

> …  When a port is compiled, it actually creates a package. The package is then installed and managed by the package management system. A binary package is the same, it has just been compiled on a different computer. 

Specifically [pkg-install(8)](https://man.freebsd.org/cgi/man.cgi?query=pkg-install&sektion=8&manpath=freebsd-ports), I guess. pkg(8) is close enough.

Reply:
> Either (a) clarify the mention of ports, or (b) don't mention ports(7) here.

This is an index. Index points to the overview on the topic. ports(7) is the topic overview for contributed applications.

Previously we only linked to ports(7). If the compromise I proposed is causing issues, we should revert to the previous behavior of only mentioning ports(7).